src/test/java/tests/pages/DashboardPage.java

```diff
@@ -0,0 +1,121 @@
+package tests.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Dimension;
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import java.util.List;
+
+import static core.utils.Timeout.TWO_SEC;
+import static org.junit.Assert.assertTrue;
+
+public class DashboardPage extends PageFactory {
+  public static final String TITLE_ALL_DASHBOARDS_XPATH = "//span[@title = 'All Dashboards']";
```

Comment:
it would be better to store the constants as a locator instead of a String. The main benefit is you can pass it as locator and do not think about type of locator and write every time By.xpath() in usage places.
you can write once: public static final String TITLE_ALL_DASHBOARDS = By.xpath("//span[@title = 'All Dashboards']"); and forget about additional static methods By.xpath By.id e.t.c